source/OptimaJet.DWKit.StarterApplication/Services/ProductService.cs

```diff
@@ -369,6 +387,8 @@ public async Task<IEnumerable<ProductActions>> GetProductActionsForProjectsAsync
             var products = await ProductRepository.Get()
                 .Where(p => ids.Contains(p.ProjectId))
                 .Include(p => p.ProductDefinition)
+                .Include(p => p.Project)
+                    .ThenInclude(pr => pr.Owner)
```

Comment:
Same here.  Do you use Project.Owner?  I didn't see it by searching through the PR.